lib/modules/manager/gomod/artifacts.ts

```diff
@@ -271,7 +273,22 @@ export async function updateArtifacts({
 
     const execCommands: string[] = [];
 
-    let args = 'get -d -t ./...';
+    let goGetDirs = './...';
+    if (config.goGetDirs) {
+      goGetDirs =
+        config.goGetDirs
+          .filter((dir) => {
+            const isValid = isValidLocalPath(dir);
+            if (!isValid) {
+              logger.debug(`invalid path in goGetDirs will be skipped: ${dir}`);
```

Comment:
```suggestion
              logger.warn({ dir }, 'Invalid path in goGetDirs';
```

Reply:
dir should be inside message, otherwise it's not visible on dashboard. 🤔